Modules/Messages/src/Messages.Infrastructure/Persistence/Database/QueryableExtensions/MessagesQueryableExtensions.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Backbone.Modules.Messages.Domain.Entities;
+using Backbone.Modules.Messages.Domain.Ids;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using Enmeshed.Tooling;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backbone.Modules.Messages.Infrastructure.Persistence.Database.QueryableExtensions;
+
+public static class MessagesQueryableExtensions
+{
+    public static IQueryable<Message> IncludeAllReferences(this IQueryable<Message> messages)
```

Comment:
There is an extension method in the BuildingBlocks.Infrastructure project called `IncludeAll` which includes all references generically. Can you use that and remove this one please (I know it isn't part of your current task)